modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb

```diff
@@ -63,9 +63,8 @@ def validate
 
         def attachments
           if params.keys.select { |key| key.include? 'attachment' }.count > 10
-            raise ::ClaimsApi::Common::Exceptions::Lighthouse::UnprocessableEntity.new(
-              detail: 'Too many attachments.'
-            )
+            raise ::ClaimsApi::Common::Exceptions::Lighthouse::UnprocessableEntity,
+                  [{ detail: 'Too many attachments.' }]
```

Comment:
do we want a source here or are you accepting the `/data/attributes` to be the source by default?

Reply:
@FonzMP  I am updating this not to share that UnprocessableEntity error method.  So that will have a source on it when that change goes in.